stripe/src/main/java/com/stripe/android/view/PaymentMethodsActivity.kt

```diff
@@ -69,6 +72,17 @@ class PaymentMethodsActivity : AppCompatActivity() {
             window.addFlags(it)
         }
 
+        viewModel.snackbarData.observe(this, Observer {
+            Snackbar.make(coordinator, it, Snackbar.LENGTH_SHORT).show()
```

Comment:
One thing to watch out for is if this activity is recreated and there is still a value in `snackbarData` you may show a snack bar for an old value that is no longer relevant: https://developer.android.com/topic/libraries/architecture/livedata#observe_livedata_objects

> To ensure that the activity or fragment has data that it can display as soon as it becomes active. As soon as an app component is in the STARTED state, it receives the most recent value from the LiveData objects it’s observing. This only occurs if the LiveData object to be observed has been set.

The way I've worked around this before is by making the `snackbarData` type nullable, and then from the view model immediately setting `null` to the value afterwards so that the observer is only triggered once.